app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+
+  def new
+    @picture = Picture.find(params[:picture_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @picture = Picture.find(params[:picture_id])
+    @picture.comments.create(comment_params)
+    redirect_to pictures_path
+  end
+
+  def comment_params
+    params.require(:comment).permit(:comment)
```

Comment:
Avoid avoid avoid naming attributes the same as the model. This means you'll be doing stuff like `comment.comment` which is seriously weird.

Better is to use general descriptors for long-form text, like `comment.text` or the more canonical `comment.body`